### PR TITLE
[close #816] Fix Question items answer table creation

### LINKED ISSUE #816: Creation of answer tables for question items stopped working
### Subject of the issue

Answer categories can be grouped together as items.
They share the same answer categories and and an overarching question.
They should be displayed like so:

__Did you ever ...__

|      | yes | no | no answer |
|------|-----|----|-----------|
| do x |    |     |             |
| do y |    |     |             |

This tabular display is no longer being produced.

### PR DESCRIPTION
**Pull request by hansendx**
* Whitespace was introduced into an if statement by an html formatter.

  + Before: `if == "thing"`
  + After:  `if == " thing"`

* This caused a failure in the creation of
  tables for answers belonging to question items

### CODE REVIEW

File: ddionrails/instruments/models/question.py [hunk 1]
```diff
@@ -87,6 +87,8 @@ def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
         """"Set id and call parents save(). """
+        # pylint: disable=invalid-name
```
Comment (afuetterer) [inline ddionrails/instruments/models/question.py]: see line 94

Comment (afuetterer) [reply ddionrails/instruments/models/question.py]: black moved the comment
`self.id = hash_with_namespace_uuid(  # pylint: disable=C0103
`
or 
`self.id = hash_with_namespace_uuid(  # pylint: disable=invalid-name`
should work.

**Comment by afuetterer** [comment]
Related issue: #816

File: ddionrails/instruments/models/question.py [hunk 2]
```diff
@@ -87,9 +87,11 @@ def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
         """"Set id and call parents save(). """
-        self.id = hash_with_namespace_uuid(
+
+        self.id = hash_with_namespace_uuid(  # pylint: disable=invalid-name
             self.instrument_id, self.name, cache=False
-        )  # pylint: disable=C0103
+        )
+        ip = ""
```
Comment (afuetterer) [inline ddionrails/instruments/models/question.py]: What is this?

Comment (hansendx) [reply ddionrails/instruments/models/question.py]: typo

**Pull request merged**